common/util_test.go

```diff
@@ -458,15 +458,6 @@ func TestParseAccelDevices_WrongDriver(t *testing.T) {
 	getPCI = originGetPCI
 }
 
-func TestParseAccelDevices_FailedDetection(t *testing.T) {
```

Comment:
@red-0ne what is the purpose of this test? It always fails, if Nvidia GPU is present.

Reply:
You are right. I think it should setup its `GetGPU`/`GetCPU`
The idea was to fail when no GPUs or appropriate PCI devices are detected (it fails now if you have a real one)
A minimal resolution would be
```go
	getGPU = func() ([]*gpu.GraphicsCard, error) {
		return []*gpu.GraphicsCard{}, nil
	}
	getPCI = func() ([]*pci.Device, error) {
		return []*pci.Device{}, nil
	}
```